modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMTemplateLocalServiceImpl.java

```diff
@@ -1039,10 +1039,24 @@ public List<DDMTemplate> search(
 		int status, int start, int end,
 		OrderByComparator<DDMTemplate> orderByComparator) {
 
+		String templateModelResourceClassName = null;
```

Comment:
Hey @MrJohn1911, do not create this temp variable. You need to apply the same pattern we have into the `DDMStructureLocalServiceImpl`. Thx

Reply:
Hey @rafaprax I tried to do it before, but if I inline this variable in the method and wrap it with the try/catch block, the searchContext is never created, changing the logic that were applied before my changes.